### ISSUE TITLE
Member Registration 

### ISSUE THREAD

**Issue by afirdousi** [angular-pakistan/ngpakistan.com#7]
Must haves:

- [ ] User should be able to register to Angular Pakistan once and RSVP to meetups with a single button click. Members should not be registering for every single meetup every single time ( which is, unfortunately, the current way of registering to our events )

- [ ] Each user will have a profile with super basic fields. 
-Name, 
-Email 1,
-Email 2 (optional),
-Phone 1,
-Phone 2 (optional),
-DOB (optional),
-Github (optional),
-Facebook (optional),
-Twitter (optional)
-Linked In (optional)

I will open a separate issue for meetup subscription which will be linked to this issue. This needs to happen first.
